### PR TITLE
preliminary summit program

### PR DESCRIPTION
**Pull request by bbrcknl**

### CODE REVIEW

File: Foundation/Summit/2024/program.html [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+    <span class="summit-abstract-affiliation">Collins Aerospace</span>
+    </td>
+  </tr>
+  <tr>
+    <td class="break" colspan="3">Break</td>
+  </tr>
+  <tr>
+    <td>10:30 - 11:00</td>
+    <td>Talk</td>
+    <td>seL4 Verification: Status and Plans<br><span class="summit-abstract-presenter">Gerwin Klein</span>,
```
Comment (june-andronick) [inline Foundation/Summit/2024/program.html:47]: Here, and elsewhere, you should have the full list of authors (as discussed Gerwin won't be presenting both of these); We should keep the full list of authors in any case, and highlight in some way the speaker when/once we know it.

File: Foundation/Summit/2024/index.html [hunk 2]
```diff
@@ -20,12 +20,12 @@ <h4>
   <ul>
     <!--li><a href="#summit-important-dates">Important Dates</a></li-->
     <li><a href="#summit-about">About</a></li>
-    <li><a class="highlight" href="#summit-registration">Latest News: Register for summit</a></li>
+    <li><a href="#summit-registration">Latest News: Register for summit</a></li>
```
Comment (june-andronick) [inline Foundation/Summit/2024/index.html:23]: Need to delete "Latest News:"

File: Foundation/Summit/2024/program.html [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+</h4>
+<h4>
+  15-17 October 2024
+</h4>
+
+<h3>
+   Program
+</h3>
+<p>
+  All times are local time in Sydney, Australia (GMT+11).
```
Comment (june-andronick) [inline Foundation/Summit/2024/program.html:21]: Maybe just add, in text, the small paragraph about social activities on Monday and bush walk on weekend.

File: Foundation/Summit/2024/program.html [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+  <!--br>
+  Session chairs in <em>italics</em>.-->
+</p>
+<table id="summit-talks">
```
Comment (june-andronick) [inline Foundation/Summit/2024/program.html:25]: I haven't had time to check all titles/times/authors. If you'd like another pair of eyes, please ask Ihor or Nick.

File: Foundation/Summit/2024/program.html [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+    <span class="summit-abstract-affiliation"></span>
+    </td>
+  </tr>
+  <tr>
+    <th colspan="3">Training day @UNSW (TBC)&emsp;18 October 2024</th>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Training</td>
+    <td>The seL4 Microkit<br><span class="summit-abstract-presenter">Ivan Velickovic</span>,
```
Comment (Ivan-Velickovic) [inline Foundation/Summit/2024/program.html:363]:
```suggestion
    <td>The Lions Operating System<br><span class="summit-abstract-presenter">Ivan Velickovic</span>,
```

File: Foundation/Summit/2024/program.html (hunk 1 above)
Comment (bbrcknl) [reply Foundation/Summit/2024/program.html:47]: I'll put the full list in the program.html. The co-authors' details are also in the speakers spreadsheet in Drive.

**Review by june-andronick — APPROVED** [review]
I sent you my small comments separately. Else fine with me. Thanks!

**Pull request merged**